src/lib/operations.ts

```diff
@@ -57,81 +61,51 @@ interface StdLibCallInfo {
  * Gather up the argument values for the Extrude command
  * to be used in the command bar edit flow.
  */
-const prepareToEditExtrude: PrepareToEditCallback =
-  async function prepareToEditExtrude({ operation, artifact }) {
-    const baseCommand = {
-      name: 'Extrude',
-      groupId: 'modeling',
-    }
-    if (
-      !artifact ||
-      !('pathId' in artifact) ||
-      (operation.type !== 'StdLibCall' && operation.type !== 'KclStdLibCall')
-    ) {
-      return baseCommand
-    }
+const prepareToEditExtrude: PrepareToEditCallback = async ({ operation }) => {
```

Comment:
Cleaned up all three below with an inverse util function to go from the sketches in the operation object to graph selections. Trying to make things as consistent as possible but still allowing for specific needs of each